### ISSUE TITLE
Deprecate legacy data page

### ISSUE THREAD

**Issue by gbinal** [GSA/gsa.github.io#17]
After #15 is closed, request the following:  
- Change the redirect for www.gsa.gov/data and gsa.gov/data to direct visitors to open.gsa.gov/data 
- Remove the content on http://gsa.gov/portal/content/181595
- Set up a 301 redirect from http://gsa.gov/portal/content/181595 to open.gsa.gov/data
  May have to submit a service request to Ken Kozen to have the 301's completed.

**Comment by gbinal** [comment]
#15 is done - I recommend moving forward with this.

**Comment by gbinal** [comment]
This is on hold while #30 is resolved.

**Comment by gbinal** [comment]
#30 is now resolved.  I believe we can move forward with this.  

@mboone, are you game to follow up with folks or want me to?

**Comment by afeld** [comment]
I'd also suggest that http://www.gsa.gov/portal/category/100370 be deprecated in favor of http://catalog.data.gov/organization/gsa-gov.

**Comment by saracope** [comment]
Closing out, will not fix here as we are retiring this site.
